### PR TITLE
Tiny tweaks in manifest internals

### PR DESCRIPTION
**Pull request by daabr**

### CODE REVIEW

File: internal/manifest/log.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package manifest
 
-import "fmt"
```
Comment (itayd) [inline internal/manifest/log.go:3]: why

File: internal/manifest/log.go [hunk 2]
```diff
@@ -11,8 +13,5 @@ type keyer interface {
 }
 
 func (l Log) For(kind string, keyer keyer) Log {
-	if kind != "" {
-		kind += " "
-	}
-	return Log(func(f string) { l.Printf("%s%v: %v", kind, keyer.GetKey(), f) })
+	return Log(func(f string) { l.Printf("%s %q: %v", kind, keyer.GetKey(), f) })
```
Comment (itayd) [inline internal/manifest/log.go:16]: if `kind` is empty you'll have an extra space in the beginning

File: internal/manifest/manifest.go [hunk 3]
```diff
@@ -26,8 +26,7 @@ type stringKeyer string
 func (s stringKeyer) GetKey() string { return string(s) }
 
 type Manifest struct {
-	Version string `yaml:"version,omitempty" json:"version,omitempty" jsonschema:"required"`
-
+	Version string   `yaml:"version,omitempty" json:"version,omitempty" jsonschema:"required"`
```
Comment (itayd) [inline internal/manifest/manifest.go:29]: daniel

File: internal/manifest/log.go (hunk 2 above)
Comment (daabr) [reply internal/manifest/log.go:16]: Yes, but in AK `kind` should never be empty.

File: internal/manifest/log.go (hunk 1 above)
Comment (daabr) [reply internal/manifest/log.go:3]: standardization of import block appearance (I wouldn't have bothered to do this, but I already touched the file for a different reason)

**Review by itayd — APPROVED** [review]

**Pull request merged**